NuGet.Docs/Consume/Command-Line-Reference.md

```diff
@@ -305,6 +305,9 @@ Deletes or unlists a package from the server. For NuGet.org, the action is to [u
     nuget delete <package Id> <package version> [API Key] [options]
 
 Specify the Id and version of the package to delete from the server.
+
+Note: In the -Source paramter, you can choose to either use https://www.nuget.org/ or https://www.nuget.org/api/v3. If you use the https://www.nuget.org/api/v2 endpoint, this command will fail. 
+
```

Comment:
Thanks. Made the requested changes.




From: Maarten Balliauw <notifications@github.com>
Reply-To: NuGet/NuGetDocs <reply@reply.github.com>
Date: Monday, April 11, 2016 at 11:24 PM
To: NuGet/NuGetDocs <NuGetDocs@noreply.github.com>
Cc: Harikrishna Menon <harikm@microsoft.com>
Subject: Re: [NuGet/NuGetDocs] Update to delete documentation (#453)


In NuGet.Docs/Consume/Command-Line-Reference.md<https://github.com/NuGet/NuGetDocs/pull/453#discussion_r59325955>:

> @@ -305,6 +305,9 @@ Deletes or unlists a package from the server. For NuGet.org, the action is to [u

>      nuget delete <package Id> <package version> [API Key] [options]

>

>  Specify the Id and version of the package to delete from the server.

> +

> +Note: In the -Source paramter, you can choose to either use https://www.nuget.org/ or https://www.nuget.org/api/v3. If you use the https://www.nuget.org/api/v2 endpoint, this command will fail.

> +

/api/v2/package is the correct one here.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/NuGet/NuGetDocs/pull/453/files/e34890ca97403fb3a02f2c3226d7f48819ba4076#r59325955>